CAAppKitBridge/Source/NSView+CAMethods.h

```diff
@@ -25,26 +25,22 @@
 */
 
 #import <AppKit/AppKit.h>
+#import <Foundation/Foundation.h>
 #import <QuartzCore/QuartzCore.h>
-#import <GNUstepBase/GSVersionMacros.h>
 
-@interface  NSView (NSViewCAmethods)
+@interface  NSView (CAmethods)
 @property BOOL wantsLayer;
 - (CALayer *) makeBackingLayer;
 
 /* GNUstep specific methods */
-- (BOOL) addCARenderer: (CARenderer*) CARenderer;
+- (BOOL) addCARenderer: (CARenderer*)renderer;
 - (BOOL) removeCARenderer;
 
-/* methods from libs-gui/Headers/AppKit/NSOpenGlView.h */
-- (void)             clearGLContext;
-- (void)             setOpenGLContext: (NSOpenGLContext*)context;
+/* methods from libs-gui/Headers/AppKit/NSOpenGLView.h */
+- (void) clearGLContext;
```

Comment:
As discussed multiple times, it is HIGHLY unlikely you need these (`-clearGLContext`, `-setOpenGLContext:`, `openGLContext`, `reshape` and `update`) and they are likely to conflict with either user code in user subclasses of `NSView`, or even with GNUstep subclasses of `NSView` such as `NSOpenGLView`.

Can you please clarify how you plan on using these methods? That is, why do you require them?

If you do need them (and I don't think you do; creation and destruction of an OpenGL context should happen only during creation and destruction of a `CARenderer`, and you can have a private method for this), can you please clarify why they can't be renamed?


<sub>(Sidenote: Looking at `-openGLContext`, I do believe you need the functionality from `-openGLContext` which creates a context on demand. I just don't think the method should be called that, nor that creation of a context should happen anytime someone asks 'what is this view's OpenGL context'. And that is, by the way, the implication from the method name.)</sub>